### ISSUE TITLE
Specifing the used standard (conformsTo)

### ISSUE THREAD

**Issue by coret** [netwerk-digitaal-erfgoed/dataset-register#859]
For the Aggregating Linked Data in Europeana MVP we need to be able to specify on which standard/model/ontology a distribution is based (eg. https://www.europeana.eu/schemas/edm/EDM.xsd). This property differs from `dct:format` / `schema:encodingFormat` where you specify the format like `application/rdf+xml`. 

A suitable DCAT property would be [dct:conformsTo](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#terms-conformsTo). As mentioned in [DCAT-AP to Schema.org Mapping](https://ec-jrc.github.io/dcat-ap-to-schema-org/#conformance), the property [schema:publishingPrinciples](http://schema.org/publishingPrinciples) would be the `nearest` Schema.org property.

We need to include this property in the requirements (recommended, cardinality 0..n), adjust the SHACL and adapt the crawler (conversion & storing).

**Comment by ddeboer** [comment]
Agreed. Note that the DKG currently discovers this property, exposing it as `void:vocabulary`, but _on the level of the dataset_, not the distribution.

**Comment by EnnoMeijers** [comment]
In the [example](https://github.com/europeana/rd-schemas-and-models/blob/main/SGoaB/DCAT-AP/v2.1.1/europeana-catalogue.ttl ) that Europeana shared with an identifier for the EDM was used including its definition: 
```
<http://www.europeana.eu/schemas/edm/>  
  a dct:Standard .
```

Should we consider using a blank node for dct:conformsTo to include also the definition or do we assume this value to always be of type dct:Standard?

**Comment by ddeboer** [comment]
According to the spec, dct:conformsTo _includes_ instead of _has_ range dct:Standard, so adding it explicitly would be meaningful.

**Comment by coret** [comment]
> Note that the DKG currently discovers this property, exposing it as void:vocabulary, 

I think [dct:conformsTo](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#terms-conformsTo) / [schema:publishingPrinciples](http://schema.org/publishingPrinciples) is broader than vocabulary (which can only be applied to linked data).

**Comment by ddeboer** [comment]
For SPARQL endpoints, the Dataset Register now automatically adds `dcterms:conformsTo <https://www.w3.org/TR/sparql11-protocol/>`.

**Comment by mzeinstra** [comment]
I see this issue is open, is this on the roadmap, do we need additional discussions on this?
